analysis/topEFT/make_jsons.py

```diff
@@ -476,122 +476,92 @@
         "histAxisName": "ST_antitop_t-channel_centralUL17",
         "xsecName": "ST_antitop_t-channel",
     },
-
     "UL17_tbarW" : {
-        "path" : "/ST_tW_antitop_5f_inclusiveDecays_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
+        "path" : "/ST_tW_antitop_5f_inclusiveDecays_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM", # NOTE: PLACEHOLDER till v9 is available
         "histAxisName": "tbarW_centralUL17",
         "xsecName": "ST_tW_antitop_5f_inclusiveDecays",
     },
-
     "UL17_tW" : {
-        "path" : "/ST_tW_top_5f_inclusiveDecays_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
+        "path" : "/ST_tW_top_5f_inclusiveDecays_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM", # NOTE: PLACEHOLDER till v9 is available
         "histAxisName": "tW_centralUL17",
         "xsecName": "ST_tW_antitop_5f_inclusiveDecays",
     },
-
     "UL17_TTGJets" : {
-        "path" : "/TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
+        "path" : "/TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8/RunIISummer20UL17NanoAODv9-106X_mc2017_realistic_v9-v1/NANOAODSIM",
         "histAxisName": "TTGJets_centralUL17",
         "xsecName": "TTGJets",
     },
-
     "UL17_TTJets" : {
-        "path" : "/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIISummer19UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
+        "path" : "/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIISummer20UL17NanoAODv9-106X_mc2017_realistic_v9-v1/NANOAODSIM",
         "histAxisName": "TTJets_centralUL17",
         "xsecName": "TT",
     },
-
-
     "UL17_WJetsToLNu" : {
-        "path" : "/WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
+        "path" : "/WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17NanoAODv9-106X_mc2017_realistic_v9-v1/NANOAODSIM",
         "histAxisName": "WJetsToLNu_centralUL17",
         "xsecName": "WJetsToLNu",
     },
-
     "UL17_WWTo2L2Nu" : {
-        "path" : "/WWTo2L2Nu_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
+        "path" : "/WWTo2L2Nu_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM", # NOTE: PLACEHOLDER till v9 is available
         "histAxisName": "WWTo2L2Nu_centralUL17",
         "xsecName": "WWTo2L2Nu",
     },
-
     "UL17_WWW_4F" : {
-        "path" : "/WWW_4F_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
-        "histAxisName": "WWW_centralUL17",
-        "xsecName": "WWW",
-    },
-
-    "UL17_WWW_4F_ext" : {
```

Comment:
Is there no longer a `WWW_ext` sample?

Reply:
Thanks for pointing this out, I probably should have mentioned it in the PR's description. I replaced this WWW sample with a v9 one, and there does not seem to be an v9 "ext" sample available (maybe it's been merged with the non ext one as you mentioned, or maybe they just have not made an "ext" one yet).

Reply:
Ok thanks. We should keep an eye on these. Looking at the `nEvents` entries, `WWW_4F` has 1.7M events (didn't change with v9) and `WWW_4F_ext` had another ~9M events.

Reply:
Ah, good point. I've added comments to these lines to remind us to check to see if "ext" samples become available.

Reply:
Thanks?